src/__snapshots__/App.snapshot.spec.tsx.snap

```diff
@@ -10,7 +10,7 @@ Array [
       className="nhsuk-width-container nhsuk-header__container"
     >
       <div
-        className="nhsuk-header__logo"
+        className="nhsuk-header__logo nhsuk-header__logo--only"
```

Comment:
May I ask what this snapshot file used for, please?

Reply:
This is part of snapshot testing - https://jestjs.io/docs/en/snapshot-testing
These snapshots help us to make sure that UI is not changed until we wanted it to.